encrypted-media/clearkey-mp4-playback-destroy-persistent-license.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
```

Comment:
Yes, none of them should need it.

Originally, I had many tests in one file and this took a long time, so we needed the long timeout. I'm trying to make them faster now and the long timeout should not be needed.